cmake/CompilerOptions.cmake

```diff
@@ -160,7 +160,7 @@ function(export_visibility target)
     # generate config file
     get_target_property(target_binary_dir ${target} BINARY_DIR)
     configure_file(
-        ${CMAKE_SOURCE_DIR}/cmake/export.hpp.in
-        ${target_binary_dir}/include/${target_short_name}/export.hpp
+        ${PROJECT_SOURCE_DIR}/cmake/export.hpp.in
+        ${CMAKE_CURRENT_BINARY_DIR}/include/${target_short_name}/export.hpp
```

Comment:
I'm wondering why `target_binary_dir` had to be replaced by `CMAKE_CURRENT_BINARY_DIR` here. Shouldn't `target_binary_dir` work even when arbor is a sub-project, but take into account any custom output location, since it depends on the target property?

Reply:
It seems to work, although I remember having issues ... weird, but maybe a dirty CMakeCache?!